examples/llm_simple_qa/README.md

```diff
@@ -0,0 +1,54 @@
+# README
+
+## Simple QA
+
+### Prepare Data
+
+The data of simple-qa example structure is:
+
+```
+.
+├── test_data
+│   └── data.jsonl
+└── train_data
+    └── data.jsonl
+```
+
+`train_data/data.jsonl` is empty, and the `test_data/data.jsonl` is as follows:
```

Comment:
As the following examples, we also need to replace questions in Chinese with those in English.